source/man/global-options.inc.in

```diff
@@ -6,11 +6,27 @@ Global :man:`doveadm(1)` *options*:
 **-D**
    Enables verbosity and debug messages.
 
+**-c** *config-file*
+   Read configuration from the given *config-file*. By default
+   it first reads config socket, and then falls back to
+   *@pkgsysconfdir@/dovecot.conf*. You can also point this to
+   config socket of some instance running compatible version.
+
+**-i** *instance-name*
+   If using multiple Dovecot instances, choose the config file based on this instance name.
+   See :dovecot_core:ref:`instance_name` setting for more information.
+
```

Comment:
Would it be possible to match the order of the options here with the SYNOPSIS?

So `[-DOkv]` but then `-c`, `-f`, `-i`, and only then `-o`. I was a bit confused that I didn't find the option where I was looking for it. Feel free to disagree and resolve this comment unchanged. :)